hikari/commands.py

```diff
@@ -136,6 +137,22 @@ class CommandOption:
         lowercase.
     """
 
+    name_localizations: undefined.UndefinedOr[typing.Mapping[typing.Union[Locale, str], str]] = attr.field(
+        default=undefined.UNDEFINED, kw_only=True
+    )
+    """A set of name localizations for this option
+
+    This will be `builtins.None` if not provided.
+    """
+
+    description_localizations: undefined.UndefinedOr[typing.Mapping[typing.Union[Locale, str], str]] = attr.field(
+        default=undefined.UNDEFINED, kw_only=True
+    )
+    """A set of description localizations for this option
+
+    This will be `builtins.None` if not provided.
+    """
```

Comment:
These should never be either undefined or None. Instead they should default to an empty dict

Reply:
mypy gives `Incompatible types in assignment` error when I do this. Should I ignore the error?